drivers/led_strip/Kconfig.ws2812b_sw

```diff
@@ -26,7 +26,7 @@ config WS2812B_SW_NAME
 
 config WS2812B_SW_GPIO_NAME
 	string "GPIO port that the LED strip is connected to"
-	default GPIO_NRF5_P0_DEV_NAME if SOC_FAMILY_NRF
+	default "GPIO_0" if SOC_FAMILY_NRF
```

Comment:
Could we not use a hard-coded value, here?

Reply:
It was set before to port name from Kconfig file, currently Kconfig option was removed in favor of dts configuration, there is no way to reference a dts symbol in Kconfig thats why I used hardcoded value, other option is to remove this line completely